diff --git a/Configurations/15-android.conf b/Configurations/15-android.conf
index 41ad9223e0..db555023f6 100644
--- a/Configurations/15-android.conf
+++ b/Configurations/15-android.conf
@@ -192,6 +192,7 @@ my %targets = (
         bin_lflags       => "-pie",
         enable           => [ ],
         shared_extension => ".so",
+        shlib_variant => '_3',
     },
     "android-arm" => {
         ################################################################
